frontend/lib/common-steps/ask-nyc-address.tsx

```diff
@@ -26,13 +20,14 @@ import { YesNoConfirmationModal } from "../ui/confirmation-modal";
 import { li18n } from "../i18n-lingui";
 import { t, Trans } from "@lingui/macro";
 import { MiddleProgressStepProps } from "../progress/progress-step-route";
+import { NycAddressMutation } from "../queries/NycAddressMutation";
+import { BlankNorentScaffolding } from "../queries/NorentScaffolding";
 
 const ConfirmNycAddressModal: React.FC<{
   nextStep: string;
 }> = ({ nextStep }) => {
   const addrInfo =
-    useContext(AppContext).session.onboardingStep1 ||
-    BlankOnboardingStep1V2Input;
+    useContext(AppContext).session.norentScaffolding || BlankNorentScaffolding;
```

Comment:
At what point do you imagine renaming NorentScaffolding to something a little more all-encompassing, like OnboardingScaffolding? With this PR, it does start to serve functions beyond NoRent, but I could imagine you wanting to do that in a separate follow-on PR - maybe when we deprecate all the OnboardingStep data objects?

Reply:
very soon! i attempted to do it in #2145 and failed utterly but soon.